migrations/2018_01_18_130400_rename_permissions_to_group_permissions.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+/*
+ * This file is part of Flarum.
+ *
+ * (c) Toby Zerner <toby.zerner@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+use Flarum\Database\Migration;
+
+return Migration::renameTable('permissions', 'groups_permissions');
```

Comment:
The migration file should be called rename_permissions_to_group***s***_permissions.php

Reply:
Can we please also add a foreign key for group_id?